Library/Homebrew/lock_file.rb

```diff
@@ -35,3 +35,15 @@ def create_lockfile
     @lockfile.fcntl(Fcntl::F_SETFD, Fcntl::FD_CLOEXEC)
   end
 end
+
+class FormulaLock < LockFile
+  def initialize(name)
+    super("#{name}.formula")
```

Comment:
This is just a cosmetic change, so we can easily see whether a lock file is for a Formula or for a Cask.